ntp-proto/src/nts_record.rs

```diff
@@ -217,33 +254,23 @@ impl NtsRecord {
 
     #[allow(unused_variables)]
```

Comment:
Is this one still necessary?

Reply:
It is, but I've narrowed the attr using cfg-attr in d58d7086c70012189c25a9a869d0c5715cd03191